src/chigame/games/admin.py

```diff
@@ -0,0 +1,6 @@
+from django.apps import apps
+from django.contrib import admin
+
```

Comment:
![Image](https://github.com/uchicago-cs/chigame/assets/86740254/cf4e6c46-c63e-4e4a-9982-9c6be5917513)

As of now, your code doesn't have the names of models displayed in the admin interface. Please add an appropriate `__str__` method to the `Game` model such that the admin interface is more readable/useable.

Reply:
Thank you for the feedback! I added a __str__ method to the Game model and customized the game model in the admin interface in the latest commit. Please let me know what you think

Reply:
In the admin interface, each game is listed with its name as well as min/max number of players. I also added a search bar, so games are searchable by their name

Reply:
FYI this search features is somewhat limited, only matching the very beginning of titles and seemingly not working for spaces:

Not that this is a huge problem that needs to be adjusted, but FYI. this could be a task to make the admin search bar better.

<img width="501" alt="image" src="https://github.com/uchicago-cs/chigame/assets/86740254/ac3314a2-d78c-4665-9b35-b7fb54b63fbb">
<img width="579" alt="image" src="https://github.com/uchicago-cs/chigame/assets/86740254/d5ef5cd2-ab50-4eb2-b966-bb3e0963833b">
<img width="598" alt="image" src="https://github.com/uchicago-cs/chigame/assets/86740254/d18f9ff8-0be9-45af-a859-b872a02e31d0">
<img width="583" alt="image" src="https://github.com/uchicago-cs/chigame/assets/86740254/1c4777a8-62d3-44b6-b951-3c19d997f8fb">
<img width="574" alt="image" src="https://github.com/uchicago-cs/chigame/assets/86740254/22868923-d375-46d1-a67a-b10601539ada">